pkgs/applications/science/astronomy/stellarium/default.nix

```diff
@@ -1,22 +1,29 @@
 { stdenv, fetchurl, cmake, freetype, libpng, mesa, gettext, openssl, perl, libiconv
-, qtscript, qtserialport, qttools
+, qtscript, qtserialport, qttools, makeQtWrapper
 }:
 
 stdenv.mkDerivation rec {
-  name = "stellarium-0.14.2";
+  name = "stellarium-${version}";
+  version = "0.14.3";
```

Comment:
I've just gotten the impression that having a separate version field (even if not otherwise used) is customary so I did that change out of habit. I have no problem changing it back :-)